test/stdcpp/Makefile

```diff
@@ -0,0 +1,19 @@
+include ../common.mak
+
+TESTS:=array
+
+.PHONY: all clean
+all: $(addprefix $(ROOT)/,$(addsuffix .done,$(TESTS)))
+
+$(ROOT)/%.done : $(ROOT)/%
+	@echo Testing $*
+	$(QUIET)$(TIMELIMIT)$(ROOT)/$* $(RUN_ARGS)
+	@touch $@
+
+$(ROOT)/%: $(SRC)/%_test.d $(SRC)/%.cpp
+	mkdir -p $(dir $@)
+	$(QUIET)$(CC) $(CFLAGS) -std=c++11 -c -o $(ROOT)/$*_cpp.o $(SRC)/$*.cpp
```

Comment:
It would seem so.

Reply:
TBH I'm surprised it didn't barf at the `-std=c++11`, though it is still clang hiding underneath anyway.